common/changes/@itwin/viewer-react/pankhur94-pnpm-overrides-fix_2025-07-30-22-21.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "adding jsdom as devDep",
```

changelogs should be user facing
```suggestion
      "comment": "",
```